collection/rancher/v2.x/windows-log-collector/README.md

```diff
@@ -0,0 +1,60 @@
+# rancher-windows-log-collector
+Windows log collector for Rancher Windows Worker Nodes
+
+
+## How To Run the Script
+
+```ps1
+Set-ExecutionPolicy Bypass
+Start-BitsTransfer https://raw.githubusercontent.com/rosskirkpat/rancher-windows-log-collector/master/log-collect-beta.ps1
+.\log-collect-beta.ps1
+```
+
+### Expected output
+#### Note: The `Unable to Collect Windows Firewall information` error is expected if it there are not Domain specific firewall rules pushed out by the customer
+
+```ps1
+ .\log-collect-beta.ps1
```

Comment:
Needs to be corrected to `.\win-log-collect.ps1`

Reply:
fixed this